Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -519,11 +600,11 @@ public static List<TreeViewItemData<ActionOrBindingData>> GetActionsAsTreeViewDa
                         if (isVisible)
                             bindingItems.Add(new TreeViewItemData<ActionOrBindingData>(GetIdForGuid(inputBindingId, idDictionary),
                                 new ActionOrBindingData(false, GetHumanReadableBindingName(serializedInputBinding, state.selectedControlScheme, controlSchemes), actionMapIndex,
-                                    false, GetControlLayout(serializedInputBinding.path), serializedInputBinding.indexOfBinding)));
+                                    false, false, GetControlLayout(serializedInputBinding.path), serializedInputBinding.indexOfBinding)));
                     }
                 }
                 actionItems.Add(new TreeViewItemData<ActionOrBindingData>(GetIdForGuid(actionId, idDictionary),
-                    new ActionOrBindingData(true, action.name, actionMapIndex, false, action.expectedControlType), bindingItems.Count > 0 ? bindingItems : null));
+                    new ActionOrBindingData(true, action.name, actionMapIndex, false, false, action.expectedControlType, actionIndex: action.wrappedProperty.GetIndexOfArrayElement()), bindingItems.Count > 0 ? bindingItems : null));
```

Comment:
Might be worth doing named parameters, at least for all these bools, going into ActionOrBindingData? Just to make things a bit clearer :)

Reply:
good point - added in [8031068](https://github.com/Unity-Technologies/InputSystem/pull/1831/commits/8031068a041df18b4715ca4b47ff5e10066529f5)